source/includes/lead_event_feed_service/_overview.md

```diff
@@ -1,3 +1,4 @@
-## Lead Events [LEFSe] [Calls, Chat, Email, Web Events]
+## Lead Events
+#### [LEFSe] [Calls, Chat, Email, Web Events]
```

Comment:
Not sure why the square brackets.  Or even what this line is for.

Reply:
Not sure either

Reply:
Removed it